Components/Badge/badge.css

```diff
@@ -0,0 +1,261 @@
+* {
+    box-sizing: border-box;
+    font-family: sans-serif;
+}
+
+:root {
+    /* badge-on-text */
+    --text-badge-bg-color: transparent;
+    --text-badge-border: 1px dashed black;
+    --badge1-bg-color: #bae6fd;
```

Comment:
You can use variables for this.

Reply:
These are variables only, you can check the live preview once to see i have provided them to user. The link is above.

Reply:
Can you explain what you are asking me to do

Reply:
Okay great, you don't need to add something.

Reply:
Ok